README.md

```diff
@@ -2,82 +2,56 @@
-argument.
-The "gct" suffix will be added if you do not add it yourself.
-Be sure to either have the script and input file in your current working
-directory, or put type out the full directory path for the script and/or input
-file. eg `/users/Bob/Desktop/create_gct_file.R`
-
-#### CLS format file creation
-CLS formatted files tell information regarding the groups or phenotype of the
-different samples and are necessary for doing gene expression differential
-analysis using GenePattern's notebooks. If you've already created a gct format
-file from your data, you can create a a CLS format using GenePattern's online
-[CLSFileCreator](http://software.broadinstitute.org/cancer/software/genepattern/modules/docs/ClsFileCreator/4)
-
-# Table of Contents  
+Each directory in this repository is a set of example workflows to follow.
+See the list of `Example Workflows` below for list of the analysis topics.
+
+*Each directory is its own set of example workflows and contains:*
+
+- A`README` that introduces you to the concepts and example analyses in that module.
+- Example dataset(s) in the `data/` folder
+- An R markdown (`Rmd`) file that you can use in RStudio to run the analysis
+- An R notebook (`nb.html`) file that is the resulting output of the Rmd file
+but in a prettier format.
+- A `results` and/or `plots` folder(s) that contains the output of the analyses.
+
+As you get more comfortable with the examples, we encourage you to apply these
+example workflows to your own refine.bio data.
+After downloading a refine.bio dataset, you can analyze them in these examples
+by copying and pasting your data and metadata TSV files from your refine.bio
+download file into the respective `data/` folder for that module.
+You'll need to change any directory paths in the notebook to have the name of
+your own dataset.
+You will likely have to alter other steps of the examples, particularly those
+having to deal with the metadata.
+
+## General requirements for the example workflows
+
+This module requires you to install the following software to run examples yourself:
+
+* [**R**](https://cran.r-project.org/)
+* [**Bioconductor**](https://bioconductor.org/install/)
+* [**tidyverse**](https://www.tidyverse.org/)
+* [**RStudio**](https://www.rstudio.com/products/RStudio/) for working with [R Notebooks](https://bookdown.org/yihui/rmarkdown/notebook.html).
+* [**devtools**](https://cran.r-project.org/web/packages/devtools/readme/README.html)
+will be required for installing some packages from GitHub. We recommend installing `devtools` from CRAN.
+
+These requirements can be installed by following the instructions at the links
+above.
+The example R Notebooks are designed to check if additional required packages
+are installed and will install them if they are not.
+Each example workflow directory will include further instructions for how to
+follow along with the examples.
+
+## Example workflows in this repository  
```

Comment:
I think we may want to reorder these a bit: 1) clustering data 2) differential expression analyses 3) validate differential expression analyses 4) pathway analysis 5) dimension reduction 6) annotation using Ensembl IDs 7) ortholog mapping 8) quantile normalizing your own data

I also think we should change the text for some of these
`Annotation using Ensembl IDs` -> `Converting from Ensembl gene IDs to another identifier`
`Validate differential expression` -> `Validating differential expression results using refine.bio data`